app/src/test/java/org/fossasia/susi/ai/helper/CredentialHelperTest.kt

```diff
@@ -0,0 +1,17 @@
+package org.fossasia.susi.ai.helper
+
+import org.junit.Assert
+import org.junit.Test
+import timber.log.Timber
+
+class CredentialHelperTest {
+    @Test
+    fun testSignupPassword() {
+        Timber.d("running password test..")
+        Assert.assertEquals(false, CredentialHelper.isPasswordValid("123345602"))
+        Assert.assertEquals(false, CredentialHelper.isPasswordValid("123345602@"))
+        Assert.assertEquals(false, CredentialHelper.isPasswordValid("123345602a"))
+        Assert.assertEquals(false, CredentialHelper.isPasswordValid("123345602@a"))
+        Assert.assertEquals(true, CredentialHelper.isPasswordValid("123345602@As"))
```

Comment:
Use assertFalse and assertTrue. Remove Timber log

Reply:
Made the suggested changes